pandas/core/strings.py

```diff
@@ -778,6 +778,8 @@ def scalar_rep(x):
     else:
 
         def rep(x, r):
+            if isinstance(x, libmissing.NAType):
```

Comment:
do we need the same change for scalar_rep? (and a test that hits it)?

Reply:
If sequence passed to repeat has None it is handled...have added a test for it

Reply:
instead of isinstance, just use `if x is NA`

Reply:
Thanks for the suggestion. Have updated.